README.md

```diff
@@ -835,6 +835,7 @@ This list is the result of Pull Requests, reviews, ideas and work done by 1100+
   * [searchly.com](http://www.searchly.com/) — Free 2 indices and 20 MB storage
   * [pagedart.com](https://pagedart.com/) - AI search as a service the free tier includes 1000 Documents, 50000 searches. Larger free tiers are possible for worthwhile projects.
   * [Meilisearch](https://www.meilisearch.com/) - An open-source, lightning-fast, and hyper-relevant search engine that fits effortlessly into your apps, websites, and workflow. Free plan includes 100K documents and 10K searches/month.
+  * [OramaSearch](https://www.oramasearch.com/) - A resilient, innovative and open-source search experience to achieve seamless integration with your infrastructure and data.
```

Comment:
It appears this is not available yet?

Reply:
It's available! You can check the docs and the package (https://www.npmjs.com/package/@orama/orama). Right now, it's an in-memory solution, in the future it's going to be cloud as well :)